src/test/java/org/commcare/formplayer/tests/BaseTestClass.java

```diff
@@ -713,6 +714,7 @@ <T> T getDetails(String[] selections, String testName, String locale, QueryData
         sessionNavigationBean.setQueryData(queryData);
         sessionNavigationBean.setIsShortDetail(isShortDetail);
         sessionNavigationBean.setIsRefreshCaseSearch(isRefreshCaseSearch);
+        sessionNavigationBean.setIsClickableIcon(isClickableIcon);
```

Comment:
what is the purpose of this ? I don't think FP cares whether a request is clickable icon or not.

Reply:
The purpose is to distinguish between get_details requests from clickable icons and other sources. That way we are only removing cases from the ui when clickable icons remove them from the case list. In other cases the 404 is displayed.

Reply:
>I think that it can be handled on Web Apps instead without needing to make a FP change for it.

There may be a way to do this. @MartinRiese any thoughts?

>Doesn't web apps know whether the response it's handling is for clickable icons request ?

As far as I can tell, it can't tell that from the variables it currently has access to.

Reply:
I removed the flag and set something up on the HQ side that serves the same function.